tests/conftest.py

```diff
@@ -29,30 +32,59 @@
     )
 
 
-# A fixture only used for testing mlflow-trace package integration.
-@pytest.fixture(autouse=IS_TRACING_SDK_ONLY)
-def remote_backend_for_tracing_sdk_test(monkeypatch):
-    # Check remote backend is running or not
+@pytest.fixture(autouse=IS_TRACING_SDK_ONLY, scope="session")
+def remote_backend_for_tracing_sdk_test():
+    """
+    A fixture to start a remote backend for testing mlflow-tracing package integration.
+    Since the tracing SDK has to be tested in an environment that has minimal dependencies,
+    we need to start a tracking backend in an isolated uv environment.
+    """
+    # Use fixed port for simplicity
+    port = 5000
+    # Start a remote backend to test mlflow-tracing package integration.
+    with tempfile.TemporaryDirectory() as temp_dir:
+        with subprocess.Popen(
+            [
+                "uv",
+                "run",
+                "--with",
+                "mlflow",
+                "--python",
+                "3.9",
+                "mlflow",
+                "server",
+                "--port",
+                "5000",
+            ],
+            cwd=temp_dir,
+        ) as process:
+            print("Starting mlflow server on port 5000")  # noqa: T201
+            try:
+                for _ in range(30):
+                    try:
+                        response = requests.get(f"http://localhost:{port}")
+                        if response.ok:
+                            break
+                    except requests.ConnectionError:
+                        print("MLflow server is not responding yet.")  # noqa: T201
+                        time.sleep(1)
+
+                mlflow.set_tracking_uri(f"http://localhost:{port}")
```

Comment:
```suggestion
                mlflow.set_tracking_uri(f"http://localhost:{port}")
                yield f"http://localhost:{port}"
```

I think `yield` is missing.

Reply:
ahh that seems to be the reason of those errors